src/openshift/openshiftItem.ts

```diff
@@ -55,6 +55,18 @@ export abstract class OpenShiftItem {
         return (validator.matches(value, '^[a-z]([-a-z0-9]*[a-z0-9])*$')) ? null : message;
     }
 
+    static hideTokenpassword(value: string) {
+        const regex = /--token\s*=\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ?  value.replace(tokenRegex[1], '**********') : value;
+    }
+
+    static hideCredentialPassword(value: string) {
+        const regex = /-p\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
```

Comment:
This regex is not going to match all the possible characters in a password, spaces for example will throw it off. I'd be in favour of something more universal and simpler, like
```typescript
/--token\s*=\s*(.*)\s/
```
That way it matches anything until the last whitespace that separates the next flag.